CMSgov/mdct-mcr
# fix nested field validation

## Description

Nested field validation was broken because the concatenation of parentFieldId was happening over and over again on every re-render of the form (which with react-hook-form is a lot).

![Screen Shot 2022-10-20 at 20 10 22](https://user-images.githubusercontent.com/27705928/197197125-fc49e071-a6ee-46be-9a09-c718b9c6c29f.png)

I'm not convinced this is the best way to do this or solve it. It seems like an inappropriately naive guard to add, and I wonder if we should instead be adding the parent field id manually ourselves, or doing it at some other time, say during initial template ingestion, so it is not repeated. What are your thoughts fellow codewrites?

### How to test

Well first check out how nested field validation is broken in main. Then see how it's not broken in this branch.

### Changed Dependencies

n/a

## Code author checklist

## Reviewer checklist (two different people)
